RecoLocalCalo/EcalRecAlgos/src/PulseChiSqSNNLS.cc

```diff
@@ -410,8 +411,8 @@ bool PulseChiSqSNNLS::NNLS() {
     
     //adaptive convergence threshold to avoid infinite loops but still
     //ensure best value is used
-    if (iter%50==0) {
-      threshold *= 10.;
+    if (iter % 16 == 0) {
+      threshold *= 2;
```

Comment:
@bendavid 
please confirm that this more gradual convergence speedup makes sense.
Is there some special sample where this becomes important?

The performance log at https://patatrack.web.cern.ch/content/ecal-multifit-hlt-optimization has
" Note that in our tests, the loop never used more than 11 iterations anyway."
Which may mean that this was not tested on a sample where this can be important.